src/com/chrome/apis/bower.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "chrome-experiments-framework",
```

Comment:
This is supposed to be the machine-readable name. I've changed it to `chrome-origin-trials`.